p5js/00_ImageClassification_MultipleImages/assets/data.json

```diff
@@ -0,0 +1 @@
+{"images": ["image_0004.jpg", "image_0005.jpg", "image_0006.jpg", "image_0007.jpg", "image_0008.jpg", "image_0009.jpg", "image_00092.jpg", "image_0010.jpg", "image_00102.jpg", "image_00103.jpg", "image_0011.jpg", "image_00112.jpg", "image_0012.jpg", "image_00122.jpg", "image_0013.jpg"]}
```

Comment:
This is probably silly, but I wonder if the python script could "prettify" the JSON so it looks more like. the following?

```json
{
   "images":[
      "image_0004.jpg",
      "image_0005.jpg",
      "image_0006.jpg",
      "image_0007.jpg",
      "image_0008.jpg",
      "image_0009.jpg",
      "image_00092.jpg",
      "image_0010.jpg",
      "image_00102.jpg",
      "image_00103.jpg",
      "image_0011.jpg",
      "image_00112.jpg",
      "image_0012.jpg",
      "image_00122.jpg",
      "image_0013.jpg"
   ]
}
```